Source/LinuxParking.API/Controllers/ParkingController.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using AutoMapper;
+using LinuxParking.API.Domain.Interfaces.Services;
+using LinuxParking.API.Domain.Models;
+using LinuxParking.API.Domain.Resources;
+using LinuxParking.API.Extensions;
+using LinuxParking.API.Utils;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LinuxParking.API.Controllers
+{
+    [Route("/api/station/{stationId}/spot/{spotId}/[controller]")]
+    public class ParkingController : Controller
+    {
+        private readonly IParkingService _parkingService;
+        private readonly IMapper _mapper;
+
+        public ParkingController(IParkingService parkingService, IMapper mapper)
+        {
+            _parkingService = parkingService;
+            _mapper = mapper;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateAsync([FromRoute] int stationId, [FromRoute] int spotId, [FromBody] CreateParkingResource resource)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState.GetErrorMessages());
+            }
+
+            var parking = _mapper.Map<CreateParkingResource, ParkingStatus>(resource);
+
+            parking.CustomerID = HttpContext.User.Claims.First(h => h.Type == "Id").Value;
+            parking.ArrivalTime = DateTime.Now;
+            parking.SpotID = spotId;
+            parking.StationId = stationId;
+
+            var res = await _parkingService.SaveAsync(parking);
+
+            if (!res.Success)
+            {
+                return BadRequest(res.Message);
+            }
+
+            var parkingResource = _mapper.Map<ParkingStatus, ParkingResource>(res.ParkingStatus);
+            return Created("", parkingResource);
+        }
+
+
+        [HttpGet]
+        public async Task<IActionResult> GetAsync([FromRoute] int stationId, [FromRoute] int spotId)
```

Comment:
Man förstår vad metoden gör om man ser att den ligger i ParkingController men man kanske ska byta namn på den för att göra det extra tydligt med vad metoden gör för något.
```suggestion
        public async Task<IActionResult> GetParkingAsync([FromRoute] int stationId, [FromRoute] int spotId)
```